docs/Unlock.md

```diff
@@ -56,9 +56,13 @@ This is recommended for users who don't fear a few bugs and want the most out-of
 !!! info "Recovery options"
     Since version `9.5.60` this mod is compatible with the same *optimal* bank plan suggested in this wiki. Please, read Recovery page carefully and get familiar with recovery strategies available when this special planning is in order. The easy and reliable Recovery for optimal bank plan users will also be available for you.
 
-!!! info "Different RTFD behavior"
+!!! warning "Different RTFD behavior"
     Any RTFD attempt will not uninstall this mod completely. Since version `9.5.60` you will need to repeat RTFD two times in a row to bring back the gateway to the same state a normal RTFD would.
 
+!!! warning "Issues with Telstra variants"
```

Comment:
This kind of info is out-of-scope here. It refers to a bug and bugs do not exist over time. Also the affected devices are not in the mod's official support list, so to make it general I would rather warn about generic potential undefined issues when installed into  not officially supported devices.